lightning-tag/data-collection.md

```diff
@@ -29,3 +48,51 @@ behaviors | New or existing behaviors as type/value pairs, where the type is a s
 ruleBuilder	| Custom keys and values to be used for the Rule Builder tool within the DMP | data: {<br/>&nbsp;&nbsp;ruleBuilder: {<br/>&nbsp;&nbsp;&nbsp;&nbsp;article_tags: ['food', 'in the news'],<br/>&nbsp;&nbsp;&nbsp;&nbsp;article_title: ['Todays Headline'],<br/>&nbsp;&nbsp;&nbsp;&nbsp;article_author: ['Bob Roberts'],<br/>&nbsp;&nbsp;},<br/>},
 thirdParty | An identifier to associate with the current browser, typically to enable server side data transfer. Your Lotame representative will provide the namespace value as necessary for your implementation. | data: {<br/>&nbsp;&nbsp;thirdParty: {<br/>&nbsp;&nbsp;&nbsp;&nbsp;namespace: 'FAKE',<br/>&nbsp;&nbsp;&nbsp;&nbsp;value: '123456789101112131415'<br/>&nbsp;&nbsp;},<br/>},
 sha256email	| The current users email address, first lower-cased, trimmed of whitespace, then hashed using SHA256	|data: {<br/>&nbsp;&nbsp;sha256email: 'lowercase_no_whitespace_sha256_hashed_email'<br/>},
+
+## Passing Data outside of Page Load
+
+### Page Interactions
+
+There are times when a Lotame Client wants to pass data to Lotame based on interactions with a page that you customer makes after the page load. For example, interactions with a video player. To do this, you can use the Lightning Tag `collect()` method to send that data even after the config data collection happens as described above.
```

Comment:
This entire paragraph can probably be replaced with "You can use the Lightning Tag `collect()` method to send data from custom events that cannot be handled through standard Lotame data collection rules at any point after the Lightning Tag has loaded."

Reply:
👍